### PR TITLE
Add Gossip Script for PTR Beastmaster

### PR DESCRIPTION
**Pull request by insunaa**
This PR adds the Gossip Menu for the Beastmaster PTR NPC, which allows the player to Stable a pet, receive a new pet or unlearn all of the pet's spells.

- [ ] Please check for gossip_menu id collisions, as I haven't been able to find the correct way to generate these.

### CODE REVIEW

File: Updates/0236_ptr_beastmaster_script.sql [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
...ctPointsPerComboPoint1`, `EffectPointsPerComboPoint2`, `EffectPointsPerComboPoint3`, `SpellVisual`, `SpellIconID`, `ActiveIconID`, `SpellPriority`, `SpellName`, `SpellName2`, `SpellName3`, `SpellName4`, `SpellName5`, `SpellName6`, `SpellName7`, `SpellName8`, `Rank1`, `Rank2`, `Rank3`, `Rank4`, `Rank5`, `Rank6`, `Rank7`, `Rank8`, `ManaCostPercentage`, `StartRecoveryCategory`, `StartRecoveryTime`, `MaxTargetLevel`, `SpellFamilyName`, `SpellFamilyFlags`, `MaxAffectedTargets`, `DmgClass`, `PreventionType`, `StanceBarOrder`, `DmgMultiplier1`, `DmgMultiplier2`, `DmgMultiplier3`, `MinFactionId`, `MinReputation`, `RequiredAuraVision`, `IsServerSide`, `AttributesServerside`) VALUES (24722, 0, 0, 0, 0, 0, 142606592, 0, 128, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 101, 0, 0, 10, 10, 0, 0, 0, 0, 0, 0, 13, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, -1, 0, 30, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 999999, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 455, 0, 50, 'Happy Pet', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 0, 133, 0, 0, 0, 0, 0, 0, 0, -1, 1, 1, 1, 0, 0, 0, 1, 0);
```
Comment (AnonXS) [inline Updates/0236_ptr_beastmaster_script.sql:87]: belongs in core repo

Comment (insunaa) [reply Updates/0236_ptr_beastmaster_script.sql:87]: Where in the core repo would you put it?
I've checked the `Spell.sql`, `spell.sql` and `scriptdev2.sql` files and none of them seem to be the correct place to add a new spell. Please advise :)

Comment (killerwife) [reply Updates/0236_ptr_beastmaster_script.sql:87]: Sql, base, dbc, cmangos fixes, spell sql, tbc section. Whats the source for the spell

Comment (insunaa) [reply Updates/0236_ptr_beastmaster_script.sql:87]: The source is 1.x.x, where this spell existed in this form, but *may* have been removed in 2.x.x.
This particular implementation was given to me by an anonymous friend who may or may not have posted in this very thread...

I'm not sure where the original from the 1.x.x client comes from tho.

Comment (AnonXS) [reply Updates/0236_ptr_beastmaster_script.sql:87]: IsServerSide is flagged as 1 so it should be cmangos classic internal.

**Pull request closed without merging**